cellprofiler/modules/medianfilter.py

```diff
@@ -36,8 +36,8 @@ def create_settings(self):
             value=3,
             minval=0,
             doc="""
-                Patch size for computing the median filter. Must be odd. Use a window with a small size to
-                remove small elements of noise. A larger window will remove larger elements of noise at the
+                Dimension in each diameter for computing the median filter. Must be odd. Use a window with a small size to
+                remove small scales of noise. A larger window will remove larger scales of noise at the
```

Comment:
"scales of noise" could be confusing but I'm not sure a good alternative. Maybe "remove noise of small size", etc?